.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,7 @@
+
+
+### Pull Request checklist
+<!-- Before submitting the PR, please address each item -->
+- [ ] This PR includes thorough **tests** or an explanation of why it does not
+- [ ] This PR includes a **CHANGELOG entry** or does not need one
+- [ ] The tests are passing after this PR (`./gradlew test`; consider a pre-push hook)
```

Comment:
Optional nit: `this PR ('./gradlew test'; consider a pre-push hook)` -> `this PR ('./gradlew test') // Consider adding a pre-push hook and removing this check`

Reply:
We don't have CI so this check would only get removed after that happens – that request is for devs to create their own relevant pre-push hooks.

Reply:
Correct me if I'm wrong, but this is intended to be shared between teams and not just Show, right?

Reply:
Yes, this automation repository is meant to be used across for most Mozilla Android projects. However, I haven't taken the time to set up CI on this automation repository so we can't remove the test checklist item until then.